lib/pub_cache.dart

```diff
@@ -34,6 +34,12 @@ class PubCache {
   List<Application> _applications;
   List<PackageRef> _packageRefs;
 
+  String get _hostedPackageDirectoryName {
```

Comment:
If a function does even trivial bits of work, I tend to use a full function (`foo()`) instead of a getter.

So, I'd prefer `String get _hostedPackageDirectoryName` ==> `String _hostedPackageDirectoryName()`

Reply:
👍 Sure thing - just pushed both this and the authors addition